addons/binding/org.openhab.binding.omnikinverter/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: OmnikInverter Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.omnikinverter;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: 
+ org.openhab.binding.omnikinverter,
+ org.openhab.binding.omnikinverter.handler
+Import-Package: 
+ javax.measure,
+ javax.measure.quantity,
+ org.apache.commons.lang,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.library.unit,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.omnikinverter,
+ org.openhab.binding.omnikinverter.handler,
```

Comment:
You should remove these imports (org.openhab.binding.omnikinverter,  org.openhab.binding.omnikinverter.handler) and move the classes in those packages to:
* org.openhab.binding.omnikinverter.internal
* org.openhab.binding.omnikinverter.internal.handler

Reply:
Is the manifest file still needed for 2.5.0 ? I can't find it in other projects anymore.

Reply:
It's nowadays automatically generated in `target/classes/META-INF` so you can safely remove it.